src/Workspaces/Remote/Core/Storage/RemotePersistentStorageLocationService.cs

```diff
@@ -8,8 +8,7 @@
 
 namespace Microsoft.CodeAnalysis.Remote.Storage
 {
-    [ExportWorkspaceService(typeof(IPersistentStorageLocationService)), Shared]
-    [Export(typeof(RemotePersistentStorageLocationService))]
+    [ExportWorkspaceService(typeof(IPersistentStorageLocationService), layer: SolutionService.WorkspaceKind), Shared]
```

Comment:
That would make this clearer as well.  Because it would say "layer: ...RemoteWorkspace".

--

Also, shoudl this workspace kind just be in our standard well known set?